packages/tsconfig-reference/README.md

```diff
@@ -36,3 +36,17 @@ yarn workspace tsconfig-reference run generate-json
 
 yarn workspace tsconfig-reference run generate-markdown
 ```
+
+You can validate any codeblocks which use twoslasher via the script:
+
+```sh
+yarn workspace tsconfig-reference run test
+
+# or to just run the linter without a build
+yarn workspace tsconfig-reference run lint
+
+# or to just one one linter
+yarn workspace tsconfig-reference run lint resolveJson
+```
+
+You can debug twoslash by setting the environment var `DEBUG="*"` in all of these too.
```

Comment:
I don't think it works that way with windows, which is why it's worded like this

Reply:
If it's important, I'd add some "(use `set` on Windows)" instead of leaving it obscure.

(But I'm guessing that it's not too important given WSL or the other 100 ways of getting a good shell on windows...)